tests_metricflow/snapshots/test_cumulative_metric_rendering.py/SqlQueryPlan/BigQuery/test_derived_cumulative_metric_with_non_default_grains__plan0_optimized.sql

```diff
@@ -8,23 +8,19 @@ FROM (
     metric_time__week
     , t2mr
   FROM (
+    -- Compute Metrics via Expressions
```

Comment:
Oh that's weird, but I guess it makes sense?

Reply:
Yeah I think they just get optimized in a different way with this change. But not an incorrect way...